openapi/p4pa-workflow-hub.openapi.yaml

```
@@ -17,7 +17,7 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/DebtPositionRequestDTO'
+              $ref: '#/components/schemas/DebtPositionDTO'
```

just give a name to the type, then reference the same name on typeMapping.
You could then remove the schema
```suggestion
              type: 'DebtPositionDTO'
```